forge-gui/res/cardsfolder/a/aragorn_company_leader.txt

```diff
@@ -0,0 +1,10 @@
+Name:Aragorn, Company Leader
+ManaCost:1 G U
+Types:Legendary Creature Human Ranger
+PT:3/3
+T:Mode$ RingTemptsYou | ValidCard$ Creature.YouCtrl+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounters | TriggerDescription$ Whenever the Ring tempts you, if you chose a creature other than CARDNAME as your Ring-bearer, put your choice of a counter from among first strike, vigilance, deathtouch, and lifelink on NICKNAME.
+SVar:TrigPutCounters:DB$ PutCounter | CounterType$ First Strike,Vigilance,Deathtouch,Lifelink | Defined$ Self
+T:Mode$ CounterPlayerAddedAll | ValidObject$ Card.Self+inRealZoneBattlefield | TriggerZones$ Battlefield | ValidSource$ You | Execute$ TrigPutCountersOther | TriggerDescription$ Whenever you put one or more counters on NICKNAME, put one of each of those kinds of counters on up to one other target creature.
+SVar:TrigPutCountersOther:DB$ PutCounter | Placer$ TriggeredSource | TriggeredCounterMap$ True | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target other creature | ValidTgts$ Creature.Other

```

Comment:
I think you need extra support to reduce the amount to 1 here compared to _Bold Plagiarist_:
https://github.com/Card-Forge/forge/blob/64625fbad20768f8cd8a140854445f37da2ad1e8/forge-game/src/main/java/forge/game/ability/effects/CountersPutEffect.java#L594